README.md

```diff
@@ -21,23 +25,31 @@ environments.
 
 ### This Repository
 
-* [C API](./modality-probe-capi): Interact with a Modality probe from C.
-* [CLI](./modality-probe-cli): The CLI used for code generation for
+- [C API](./modality-probe-capi): Interact with a Modality probe from C.
+- [CLI](./modality-probe-cli): The CLI used for code generation for
   probes and the visualization of a trace.
-* [UDP Collector](./collectors/modality-probe-udp-collector): A
+- [UDP Collector](./collectors/modality-probe-udp-collector): A
   UDP-based service that collects probes' outgoing reports.
-* [Debug Collector](./collectors/modality-probe-debug-collector): A
+- [Debug Collector](./collectors/modality-probe-debug-collector): A
   collector that uses JTAG to retrieve data from the probes' logs.
-* [Batch
-  Collector](./collectors/modality-probe-offline-batch-collector): A
+- [Batch Collector](./collectors/modality-probe-offline-batch-collector): A
   utility for converting batches of binary report blobs into log
   files.
 
+
+### Open-source `modality-probe` vs. Commercial [Modality](https://docs.auxon.io/modality/)
+This `modality-probe` repository represents a subset of Modality's toolset.
+- Instrumentation for probes and events are the same in `modality-probe` and Modality.
+-  Modality adds instrumentation for [mutators](https://docs.auxon.io/modality/reference/glossary.html#mutator), which can change system state.
+- `modality-probe` relies upon [UDP Collector](./collectors/modality-probe-udp-collector), [Debug Collector](./collectors/modality-probe-debug-collector), and [Batch Collector](./collectors/modality-probe-offline-batch-collector) to retrieve, collect, and convert data from probes' logs. Modality has a unified daemon which collects probe reports into a database.
```

Comment:
`modality-probe` relies upon the UDP Collector, Debug Collector, and Batch Collector to retrieve and analyze data from probes.

Reply:
:question: @paelionmoss  Do UDP Collector, Debug Collector, and Batch Collector really analyze data from probes, except in the sense of converting them blobs?

Reply:
As far as I understand they collect reports and synthesize them into a graph, which is what goes in the file. The open source thing doesn't do any analysis beyond that - all you can do is print out the graph. So I would say yes, building the graph is the analysis part. Can double check with @jonlamb-gh

Reply:
Right, so in that case we shouldn't say "`...to retrieve and analyze data from probes." How about:

`modality-probe` relies upon the UDP Collector, Debug Collector, and Batch Collector to retrieve data from probes and output logs.

Reply:
Per discussion, going with this.

Reply:
Not quite: these oss collectors will convert the binary report protocol bytes into jsonl entries (the `session_0_log_entries.jsonl` files referred to elsewhere), the graph construction happens in memory

Reply:
So, @jonlamb-gh , would it be sufficiently accurate to say:

`modality-probe` relies upon the UDP Collector, Debug Collector, and Batch Collector to retrieve data from probes and output jsonl entries.